language-support/ts/packages/package.json

```
@@ -24,6 +24,9 @@
     "eslint": "^6.8.0",
     "jest": "^24.9.0",
     "jest-cli": "^24.9.0",
+    "jest-mock-console": "^1.0.0",
+    "jest-websocket-mock": "^2.0.2",
+    "mock-socket": "^9.0.3",
```

We decided to not use them. We also need to update the `yarn.lock` accordingly.
```suggestion
    "jest-websocket-mock": "^2.0.2",
    "mock-socket": "^9.0.3",
```